cmd/skopeo/sync.go

```diff
@@ -240,6 +253,75 @@ func syncSourceHandler(c *cli.Context, globalDestRef types.ImageReference) (toCo
 	return
 }
 
+func syncSourceFileHandler(c *cli.Context) (toCopyList []imagesBySource, err error) {
+	var cfg sourceCfg
+	if err = cfg.Unmarshal(c.String("source-file")); err != nil {
+		return nil, fmt.Errorf("Unmarshaling failed: %s", err)
+	}
+
+	for domain, domainCfg := range cfg {
+		logrus.Infof("Processing all images for domain: %s", domain)
```

Comment:
`logrus` supports structured logging (cool when used with stuff like ELK). Change the code to something like:
```go
logrus.WithFields(log.Fields{
   "domain": domain,
}).Info("Processing all images from domain")
```